Patch to allow compilation with gcc-4.4

Notified by Daniel J. - from Debian bug tracker

http://bugs.gentoo.org/252745

--- a/switchuser.h	2008-11-11 19:40:18.000000000 +0000
+++ a/switchuser.h	2008-11-11 19:40:28.000000000 +0000
@@ -18,6 +18,7 @@
 #include <pwd.h>
 #include <grp.h>
 #include <paths.h>
+#include <cstdio>
 #include <iostream>
 #include "const.h"
 #include "cfg.h"

